dockerfile-generator.rb

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/env ruby
```

Comment:
yeah but updating the template it's more easier that the `Dockerfile` itself. It's more easy and  faster to add command or anything. At least from my experience when trying to build it this morning